docarray/array/storage/weaviate/find.py

```diff
@@ -113,6 +158,10 @@ def _filter(
 
             doc.tags['wid'] = result['_additional']['id']
 
+            if additional:
+                for add in additional:
+                    doc.tags[f'{add}'] = result['_additional'][add]
```

Comment:
Why is it important to have it in the `docs tags`? What would be an example of this meaning?